### DISCUSSION TITLE
Linux Mint is good but the Linux Mint Forum and IRC are toxic places.

### DISCUSSION THREAD

**Discussion by robsennk** [community/community#44260] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I like Linux Mint. I like the DEV Team. But I dont know if Clem and his DEV are even knowing what is happen in the forum and IRC. The forum and IRC under the direction of xenopeek is horror! He and his friend pjotr continuously scare away new users, abuse their status in the forum and generally dont cast a good light on Linux Mint. They continously attack new users and ban them away and hide all information about critics. I don’t know if Clem and the dev team even know what’s going on in the forum. An elitist bubble has formed that doesn’t let anyone in and is absolutely hostile to new users and changes with all their input. That is not the MINT PHILOSOPHY of beeing user friendly. The moderators like SMG and karlchen are also taking part in this abusing maybe cause they are scared of xenopeek. xenopeek, pjotr and all their names are poison for the project and Linux Mint and Clem and his team should urgently think about replacing them. Because word gets around what goes on in the Linux Mint forum. In computerbase and other forums, Linux Mint is downright shitstormed for the behaviour of these people towards switchers!

**Comment by ghost** [comment]
The year is 2024 and this still holds true.
The moderator "SMG" has anger management issues. She will treat any comment to her comment as a challenge to an argument, will be patronizing and disrespectful, make fun of you and if you dare to defend yourself and point out the rude behavior, she will delete the posts containing the evidence but then send you a private message warning of a ban.
This is bully behavior and someone with mental issues, yet they have been given "Global Moderator" powers. I checked her post history, every 1 of 3 posts she makes are angry and passive-aggressive, to people who are just asking for help and thoughts about their issues.
This is more than enough to alienate new users.
The forum should not serve as someone's private place to vent and new users should not be anyone's punching bag.
It is shameful this has been let to happen for so long.
